lib/ohloh_scm/adapters/svn/misc.rb

```diff
@@ -133,6 +133,19 @@ def opt_auth
 			" #{opt_password} --no-auth-cache "
 		end
 
+    # Svn root is not usable here since several projects are nested in subfolders.
+    # e.g. https://svn.apache.org/repos/asf/openoffice/ooo-site/trunk/ 
+    #      http://svn.apache.org/repos/asf/httpd/httpd/trunk
+    #      http://svn.apache.org/repos/asf/maven/plugin-testing/trunk
+    #      all have the same root value(https://svn.apache.org/repos/asf)
+    def tags
+      base_url = url.sub(/\/(trunk|branches)\/?.*$/, '').chomp('/')
```

Comment:
We have around 180 svn repositories which has tag url, so we should also take care of removing tags from the url 
```sql
540648 | https://rdfsharp.svn.codeplex.com/svn/tags/0.6.2
```